aten/src/ATen/native/cuda/CUDAJitLoops.cuh

```diff
@@ -49,8 +49,8 @@ struct JittedVecKernelCache {
   at::cuda::jit::NvrtcFunction vec1;
   at::cuda::jit::NvrtcFunction vec2;
   at::cuda::jit::NvrtcFunction vec4;
-  at::cuda::jit::NvrtcFunction vec8;
 #ifdef USE_ROCM
+  at::cuda::jit::NvrtcFunction vec8;
```

Comment:
Upstream patch is different from internal_testing branch - to be review with Jerry.
@jerrymannil